fastlane/Fastfile

```diff
@@ -399,6 +401,37 @@ platform :ios do
     )
   end
 
+  #####################################################################################
+  # generate_strings_file_for_glotpress
+  # -----------------------------------------------------------------------------------
+  # Generate the `.strings` file to be imported by GlotPress,
+  # by parsing source code for `NSLocalizedString` (using `genstrings` under the hood).
+  #
+  # Typically called by complete_code_freeze.
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane generate_strings_file_for_glotpress
+  #####################################################################################
+  lane :generate_strings_file_for_glotpress do
+    cocoapods
+
+    en_lproj = 'WordPress/Resources/en.lproj'
+    ios_generate_strings_file_from_code(
+      paths: ['WordPress/', 'Pods/WordPress*/', 'Pods/WPMediaPicker/', 'WordPressShared/WordPressShared/', 'Pods/Gutenberg/'],
+      exclude: ['*Vendor*', 'WordPress/WordPressTest/I18n.swift', 'WordPress/WordPressStatsWidgets/Views/Localization/LocalizedStringKey+extension.swift', '*/Secrets.swift'],
+      output_dir: en_lproj
+    )
+
+    # @TODO: Remove once we migrated the codebase of the widgets to use the app bundle for .strings
+    today_widget_base = 'WordPress/WordPressTodayWidget/Base.lproj'
+    ios_generate_strings_file_from_code(paths: ['WordPress/WordPressTodayWidget'], output_dir: today_widget_base)
+    share_ext_base = 'WordPress/WordPressShareExtension/Base.lproj'
+    ios_generate_strings_file_from_code(paths: ['WordPress/WordPressShareExtension'], output_dir: share_ext_base)
+    # END TODO
```

Comment:
This whole set of calls will very soon disappear – as soon as I implement Step 2 of paaHJt-2J8-p2 – which I plan to tackle right after those PRs finishing up Step 1.

But I figured because I'm not sure if Step 2 will be ready and merged in time before next code-freeze, I decided to keep that logic for now – so that the `Scripts/extract-framework-translations.swift` script (which is called before each new beta and before the final release) can still rely on those as it needs them to work (see 4️⃣  in paaHJt-2KK-p2).

Once I've migrated the whole codebase of the Widgets to make them look up their strings in the App bundle (rather than their own) – aka Step 2 planned later this week – and validated that this new way works as expected (ie widgets can indeed look up strings from a _parent_ bundle), these lines to generate the 2 `Base.lproj` (and the logic in 4️⃣ `extract-framework-translations.swift` which rely on them) won't be needed anymore and those will be removed.